src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/landing_page_template.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Landing page template'
+tags:
+translate:
+redirects:
+---
+
+Landing pages are a specialized type of page that serve as the starting pages for various New Relic products. For example, you'll see landing pages for **Application monitoring (APM)** and **Browser monitoring**. 
+
+<Callout variant="tip">
+  
+  This landing page information does not apply to the docs home page.
+  
+</Callout>
+
+
+If you need to create a new landing page, you can either copy an existing landing page, or you can modify the sample landing page shown at the bottom. Let's look at the sections you need for each landing page.
+
+## Front matter
+
+When you insert the front matter, be sure to designate the `type` as `landingPage`. Here's an example:
+
+```
+---
+title: New Relic APM
+type: landingPage
+tags:
+translate:
+redirects:
+---
+```
+
+## Introduction section
+
+After you insert the front matter, insert the introduction, which typically just includes text and an image. If you need help with inserting images, see the example at the bottom.
+
+## Tiles
+
+Tiles are a series of boxes that contain the main subject areas for your product. You should just list these in order you want them to appear, and the cascading style sheet will render them across the page.
+
+Here's an example of a tile: 
+
+```
+<LandingPageTileGrid>
+  <LandingPageTile
+    title="INSERT_YOUR_BOX_TITLE_HERE."
+    icon="fe-INSERT_THE_ICON_NAME"
+  >
+
+Insert tile content here...
+
+</LandingPageTile>
+```
+For each tile, do the following:
```

Comment:
Not sure where you want to mention it, but the `LandingPageTile` also accepts an `href` property, which accepts a URL path to link to. If you provide the `href`, this will make the whole tile a link to the specified location. You can see an example of this here: https://docs-preview.newrelic.com/docs/apis

Reply:
@jerelmiller, as I tried to explain how the URL works in `LandingPageTile`, I realized that the landing pages seem to be named "index.html" instead of a specific name. What's the rationale for that? Should I mention that in the path (for example, href="/docs/apis/getting-started/intro-apis/introduction-new-relic-apis") that users can omit the name of the file (index.html)?

Reply:
Any page that is named `index.mdx` will get a route that matches its parent folder. Its a common pattern so you can group files together in a the same folder. And you're exactly right, user's can omit the `index.html` in the URL when referencing them.

| Path | URL |
|------|-------|
| `src/content/docs/apm/index.mdx` | `/docs/apm` |
| `src/content/docs/apm/intro.mdx` | `/docs/apm/intro` |
| `src/content/docs/apm/get-started/index.mdx` | `/docs/apm/get-started` |

Does that help?